### PR TITLE
Feature/init cluster at startup

### PR DESCRIPTION
**Pull request by jsvisa**

### CODE REVIEW

**Comment by jsvisa** [comment]
@ijumps  PTAL

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ RUN apt-get update \
 ADD ./heketi /usr/bin/heketi
 ADD ./client/cli/go/heketi-cli /usr/bin/heketi-cli
 ADD ./heketi-start.sh /usr/bin/heketi-start.sh
+ADD ./heketi-init.sh /usr/bin/heketi-init.sh
 VOLUME [/etc/heketi, /var/lib/heketi]
```
Comment (ijumps) [inline Dockerfile]: `[/etc/heketi, /var/lib/heketi]` -> `["/etc/heketi", "/var/lib/heketi"]`

**Comment by ijumps** [comment]
`exec "/usr/bin/heketi --config=$config"` should be `exec "/usr/bin/heketi" "--config=$config"` in file `heketi-start.sh`

File: heketi-init.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+IFS=';' read -ra endpoints <<< "$ENDPOINTS"
+for endpoint in "${endpoints[@]}"; do
+    if grep -q "$endpoint" $record_path; then
+        echo "$endpoint already loaded; skip"
+        continue
+    else
+        echo "loading $endpoint"
+    fi
+
+    printf "{\"endpoints\": [\n\t{\"nodes\": [" > $volume_path/tmp-topology.json
```
Comment (ijumps) [inline heketi-init.sh]: `endpoints`  -> `clusters`

Comment (ijumps) [reply heketi-init.sh]: ref: https://github.com/heketi/heketi/blob/master/client/cli/go/topology-sample.json

File: Dockerfile (hunk 1 above)
Comment (jsvisa) [reply Dockerfile]: @ijumps 这个双引号现在是强制的吗？

File: heketi-init.sh (hunk 2 above)
Comment (jsvisa) [reply heketi-init.sh]: @ijumps fixed

**Comment by jsvisa** [comment]
@ijumps IMO, the previous is correct, the command and argument should be an entirety, then passed to `exec`

> exec "/usr/bin/heketi --config=$config" should be exec "/usr/bin/heketi" "--config=$config" in file heketi-start.sh

**Comment by ijumps** [comment]
LGTM

**Pull request merged**